packages/dds/shared-object-base/src/serializer.ts

```diff
@@ -14,6 +14,22 @@ import {
 
 import { RemoteFluidObjectHandle } from "./remoteObjectHandle.js";
 
+function isFluidHandle(value: unknown): value is IFluidHandle {
```

Comment:
@CraigMacomber - I'm porting back #21102  to before you added `isFluidHandle`.  Does this seem like a reasonable port of that function to you?  I meant to clear "auto-merge" and get your feedback before merging, I'll do a follow-up PR if you have any suggestions.  Thanks!